tests/functional/globals.py

```diff
@@ -1,13 +1,27 @@
 import os
 
+def get_builtin_modules():
+    module_path = os.popen("%s --version | grep \"Module Path\" | cut -d ' ' -f 3" % get_syslog_ng_binary(), 'r').read().split(".")
+    return '%s:' % module_path
+
```

Comment:
while reviewing the patches I see two separate interpretation of the term "module_path" which should be separated:

1) it is used for the default module directory as used by "make install"
2) it is used to store the list of directories where modules are searched for.

The 2nd usually includes the first, but probably the name should indicate which one we are working on:

"module_dir" should be used for installation directory
"module_path" should only be used for the colon separated list of directories.